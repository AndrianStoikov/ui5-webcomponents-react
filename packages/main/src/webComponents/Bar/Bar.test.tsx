import { render } from '@shared/tests';
import { Bar } from '@ui5/webcomponents-react/lib/Bar';
import React from 'react';

describe('Bar', () => {
  test.skip('Basic Test (generated)', () => {
    const { asFragment } = render(
      <Bar
        startContent={<span>Start Content</span>}
        middleContent={<span>Middle Content</span>}
        endContent={<span>End Content</span>}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
