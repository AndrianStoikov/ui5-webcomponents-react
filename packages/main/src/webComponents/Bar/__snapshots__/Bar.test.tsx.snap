// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar Basic Test (generated) 1`] = `
<DocumentFragment>
  <ui5-bar
    design="Header"
    ui5-bar=""
  >
    <span
      slot="endContent"
    >
      End Content
    </span>
    <span
      slot="middleContent"
    >
      Middle Content
    </span>
    <span
      slot="startContent"
    >
      Start Content
    </span>
  </ui5-bar>
</DocumentFragment>
`;
